src/environment_setup/python_package.ml

```diff
@@ -72,5 +72,5 @@ let default ~host ~run_program ~install_path () =
     create_python_tool ~host ~run_program ~install_path
       ~version:"0.0.21" (Pip, Package_PyPI "topiary");
     create_python_tool ~host ~run_program ~install_path
-      ~version:"0.3.1" (Pip, Package_PyPI "vaxrank");
+      ~version:"0.4.0" (Pip, Package_PyPI "vaxrank");
```

Comment:
My understanding was that we want to do this regardless, it'll just have more of an effect with FFPE/RiboZero samples. Was it an FFPE sample that initially motivated this change?